app/helpers/table_helper.rb

```diff
@@ -4,4 +4,8 @@ def app_table(*args, **kwargs, &block)
       govuk_table(*args, **kwargs, &block)
     end
   end
+
+  def make_not_yet_known_secondary_colour(placement)
+    placement.provider_name == I18n.t("placements.schools.placements.not_yet_known") || placement.mentor_names == I18n.t("placements.schools.placements.not_yet_known") ? "secondary-text" : ""
+  end
```

Comment:
We may be able to refactor this further by changing the argument slightely, how do you feel about something like this?

```ruby
def table_text_class(data)
  data == "Not yet known" ? "secondary-text" : ""
end

table_data_class(placement.provider_name)
table_data_class(placement.mentor_names)
```